docs/Using-Virtual-Environment.md

```diff
@@ -51,7 +46,6 @@ the latest pip version using `$ pip3 install --upgrade pip`
 1. To activate the environment execute `python-envs\sample-env\Scripts\activate`
 1. Verify pip version is the same as in the __Installing Pip__ section. In case it is not the
 latest, upgrade to the latest pip version using `pip install --upgrade pip`
-1. Install ML-Agents package using `pip install mlagents`
```

Comment:
I think this should not be deleted, otherwise the mlagents python package will not be available in the virtual environment.

Reply:
I'll fix the wording, but I don't believe the page on virtual env should cover installation steps.